command/agent/http.go

```diff
@@ -291,6 +291,10 @@ func (s *HTTPServer) registerHandlers(enableDebug bool) {
 	s.handleFuncMetrics("/v1/kv/", s.wrap(s.KVSEndpoint))
 	s.handleFuncMetrics("/v1/operator/raft/configuration", s.wrap(s.OperatorRaftConfiguration))
 	s.handleFuncMetrics("/v1/operator/raft/peer", s.wrap(s.OperatorRaftPeer))
+	s.handleFuncMetrics("/v1/operator/keyring/install", s.wrap(s.OperatorKeyringInstall))
```

Comment:
I'm thinking we can get rid of these /verb suffixes in the URLs and use HTTP verbs for everything:

GET /v1/operator/keyring -> list keys
POST /v1/operator/keyring -> Install a key
PUT /v1/operator/keyring -> use a key
DELETE /v1/operator/keyring -> remove a key

Still not super REST-y since we don't have the key in the path, but looks more REST-y and keeps the API a little smaller.